tests/test_grapple.py

```diff
@@ -1423,6 +1463,10 @@ def setUp(self):
         )
 
     def test_query_single_setting(self):
+        # This only works if there is a single Site, so let's make that true.
```

Comment:
I don't get this... Why are we restricted to one setting? Shouldn't we be able to retrieve the desired setting regardless of how many are in the database?